tests/integration/test_utils_archive_readers.py

```
@@ -295,6 +295,40 @@ def test_raises_import_error_when_py7zr_unavailable(self, tmp_path: Path) -> Non
+        fake_archive.password_protected = False
+        fake_archive.__enter__ = lambda s: fake_archive
+        fake_archive.__exit__ = MagicMock(return_value=False)
+
+        mock_py7zr = MagicMock()
+        mock_py7zr.SevenZipFile.return_value = fake_archive
+
+        archive_path = tmp_path / "test.7z"
+        archive_path.write_bytes(b"dummy")
+
+        with (
+            patch.object(archives, "PY7ZR_AVAILABLE", True),
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)